configure.ac

```diff
@@ -2473,6 +2473,13 @@ fi
         fi
     fi
 
+if test "${enable_ebpf}" = "yes" || test "${enable_unittests}" = "yes"; then
+	AC_DEFINE([CAPTURE_OFFLOAD_MANAGER], [1],[Building flow bypass manager code])
```

Comment:
indent looks off. For some reason we use 2 spaces and don't align all the way to the left currently.

Reply:
Oops my bad on tabulation. On align to the left the following lines are left indented. Should I propose a big reindent at first left ?